prov/efa/src/efa_mr.c

```diff
@@ -343,7 +403,7 @@ static int efa_mr_reg(struct fid *fid, const void *buf, size_t len,
 	md->mr_fid.fid.fclass = FI_CLASS_MR;
 	md->mr_fid.fid.context = context;
 	md->mr_fid.fid.ops = &efa_mr_ops;
-
+	md->is_new_entry_reg = false;
```

Comment:
Why efa_mr_reg() does not call efa_shm_mr_reg() and efa_mr_map_insert()?

Reply:
efa_mr_reg is called by dgram(does not support shm), rxr_inline_mr_reg and rxr_buf_region_alloc_hndlr and the current functionality did not register shm or insert the key into mr_map. Do we need this change?